### PR TITLE
EDGEV3-1030/cicd-deployment

### PR DESCRIPTION
**Pull request by ololodream**
[EDGEV3-1030]
image

### CODE REVIEW

File: .buildkite/pipeline.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+steps:
+  # unit testing (all branches)
+  - label: "Cypress tests"
+    command:
+      - "apt-get -y install libgtk2.0-0 libgtk-3-0 libgbm-dev libnotify-dev libgconf-2-4 libnss3 libxss1 libasound2 libxtst6 xauth xvfb"
```
Comment (feifanz) [inline .buildkite/pipeline.yml:5]: I recommend you create a docker image instead of install packages here, which can avoid re-install every time.

**Review by feifanz — COMMENTED** [review]
I left a comment, please have a look.

**Review by feifanz — APPROVED** [review]
LGTM! Maybe wait tony to approve before merge

**Review by tonynguyenit18 — APPROVED** [review]
LGTM

**Pull request merged**